build.gradle

```diff
@@ -40,6 +40,10 @@ File getEnvironmentScript()
 apply from: environmentScript
 apply from: "${buildScriptDirPath}/configBuildScript.gradle"
 
+def versions = [
+  'netty': '4.1.58.Final'
```

Comment:
From https://netty.io/downloads.html, netty-4.1.63.Final is recommended.

Reply:
Done!